azblob/error.go

```diff
@@ -56,7 +56,7 @@ func (e *Error) StatusCode() int {
 		logger.Sugar.Debugf("Return statusCode %d", e.statusCode)
 		return e.statusCode
 	}
-	logger.Sugar.Debugf("Return InternalServerError")
+	logger.Sugar.Debugf("Return InternalServerError: %v", e)
```

Comment:
if its already established we need this debug then fine (things we know are errors are probably fair tbg). ideally we would know why or remove

Reply:
This morning I was getting a 500 with no explanation. I added the error to see the cause which turned out to be a read timeout when communicating with Azure.
lets us keep this change.